linera-sdk/src/lib.rs

```diff
@@ -32,7 +32,6 @@ pub mod util;
 pub mod abis;
 pub mod base;
 pub mod contract;
-#[cfg(feature = "ethereum")]
```

Comment:
Are we removing the feature for good? I imagine Ethereum adds quite a lot of dependencies, so it might be good to keep the feature, to improve build times for apps that don't use it?

Reply:
I found a way to put back the feature